collect_app/src/main/java/org/odk/collect/android/utilities/FormEntryPromptUtils.java

```diff
@@ -37,6 +40,18 @@ public static String getAnswerText(FormEntryPrompt fep, Context context) {
         IAnswerData data = fep.getAnswerValue();
         final String appearance = fep.getQuestion().getAppearanceAttr();
 
+        if (data instanceof SelectMultiData) {
+            String text;
+            StringBuilder b = new StringBuilder();
+            List<Selection> values = (List<Selection>) data.getValue();
+            for (Selection value : values) {
```

Comment:
Here I would just use `for (Selection value : (List<Selection>) data.getValue()) {` insteadof creating `values` list which is used only here.

Reply:
Done